### PR TITLE
assigned gcloud and ws info to variable according to node environment

### PR DESCRIPTION
**Pull request by Sibelius343**
@chanceeakin

### CODE REVIEW

File: studioclass-backend/src/utils/gCloud.js [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
-  keyFilename: join(__dirname, '../../fleet-petal-329519-cb1e7f8f25e8.json'),
-  projectId: 'fleet-petal-329519'
-});
+const gc = process.env.NODE_ENV === 'production'
+  ? new Storage({
+    credentials: JSON.parse(process.env.GCS_KEYFILE),
+    projectId: 'fleet-petal-329519'
+    })
+  : new Storage({
+    keyFilename: join(__dirname, '../../fleet-petal-329519-cb1e7f8f25e8.json'),
```
Comment (chanceeakin) [inline studioclass-backend/src/utils/gCloud.js:10]: Consider storing things like project IDs in environment variables, which can be changed agnostic to your source code.

File: studioclass-frontend/src/index.js [hunk 2]
```diff
@@ -19,11 +19,13 @@ const authLink = setContext((_, { headers }) => {
   }
 });
 
-// ws://localhost:4000/graphql/subscriptions
-// wss://studioclass.herokuapp.com/graphql/subscriptions
+const wsUri = process.env.NODE_ENV === 'production'
+  ? 'wss://studioclass.herokuapp.com/graphql/subscriptions'
+  : 'ws://localhost:4000/graphql/subscriptions'
```
Comment (chanceeakin) [inline studioclass-frontend/src/index.js:24]: Same thing with URLs.

Storing these constants in env vars would also remove the need for a ternary operator here.

**Review by chanceeakin — COMMENTED** [review]
Small PRs are great! Left some suggestions for you on these.

Also...add `.DS_Store` to your `.gitignore`

File: studioclass-backend/src/utils/gCloud.js (hunk 1 above)
Comment (chanceeakin) [reply studioclass-backend/src/utils/gCloud.js:10]: Adding all of these keys/secrets to env vars also alleviates the need for the ternary operator here. 

This also adds to dev/prod parity: [12 factor app](https://12factor.net/)

Comment (Sibelius343) [reply studioclass-backend/src/utils/gCloud.js:10]: Yeah, you're right it does make more sense to do that. The only problem I've run into with this is that the keyFile for setting up the storage object is JSON, and I haven't been able to figure out how to use JSON in a .env file. Any suggestions?

Comment (chanceeakin) [reply studioclass-backend/src/utils/gCloud.js:10]: Hey! Env vars are read into Node as a string, but Node provides ways to convert strings to JSON (namely stringify and parse). Store the json as a string, then build the object like so:

```javascript
const storageParams = {
  keyFilename: JSON.parse(process.env.keyFilename,
  ...
}

new Storage(storageParams)
```

**Pull request merged**